examples/visualization/brain.py

```diff
@@ -0,0 +1,106 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+# Plot a brain image
+# ------------------
+#
+# First we plot a brain. This requires a directory with MRI? information.
+# MNE provides functionality to automatically retrieve standard head images.
+# In this example we use the ``sample`` data which is data from an example
+# subject being shown auditory and visual stimuli.
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, background='white', cortex='low_contrast')
+
+
+# %%
+# Add source information
+# ----------------------
+#
+# Plot source information.
+sample_dir = op.join(data_path, 'MEG', 'sample')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+stc.crop(0.09, 0.1)
+
+fmin, fmax = stc.data.min(), stc.data.max()
+for i, hemi in enumerate(('lh', 'rh')):
```

Comment:
General curiosity question here. This seems the trickiest part of working with brain. Are there plans to simplify this? I.e. add a brain.add_stc method or something?

Reply:
I haven't touched it and it's not really what I'm working on right now but seems easy enough and could definitely be refactored. Just my opinion but `add_data` seems overly general/vague. Although I know stc data is in the brain space, it is not equivalent to the brain, just a more or less sparse sampling, that's just my two cents though

Reply:
Thanks. This tutorial already greatly helped me. I will try this little snippet locally. Appreciate it!